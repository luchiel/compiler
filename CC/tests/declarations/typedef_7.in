typedef struct c d;
typedef struct c {int b;} d;
