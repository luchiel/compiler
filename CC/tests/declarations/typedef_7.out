type <int>
type <float>
type <void>
type <int*>
	pointer to <int>
type <void*>
	pointer to <void>
variable NULL of type <void*> initialized by
	{int 0}
		type <int>
function printf
	returns type <int>
	arguments
		<no args>
type <0>
	struct 0
		tag c
		variable b of type <int>
type <d>
	alias to type <0>
